### DISCUSSION TITLE
Calculation of HRR through HRRPUV

### DISCUSSION THREAD

**Discussion by Ranhill** [firemodels/fds#13057] [Q&A]
Hi, I am trying to understand the use of volume integral and volume mean to calcuate the HRR in the given volume which is 4m x 4m x 4m (volume is 64m3). I made following input file:

 -------------User Section (not generated by PyroSim)-------------

&DEVC XB= 0, 4, 0, 4, 0, 4, QUANTITY= "HRRPUV", ID= "HRR_in_full_volume", SPATIAL_STATISTIC = "VOLUME INTEGRAL"/

&DEVC XB= 0, 4, 0, 4, 0, 4, QUANTITY= "HRRPUV", ID= "HRRPUV_in_full_volumne_by_volume_mean", SPATIAL_STATISTIC = "VOLUME MEAN"/

&DEVC XB= 0, 4, 0, 4, 0, 4, QUANTITY= "HRRPUV", ID= "HRRPUV_in_full_volume_direct"/

--------------------PyroSim-generated Section--------------------

&HEAD CHID='e0'/
&TIME T_END=15.0/
&DUMP DT_RESTART=300.0, DT_SL3D=0.25/

&MESH ID='Mesh01', IJK=20,20,20, XB=0.0,4.0,0.0,4.0,0.0,4.0/

&REAC ID='Reaction1',
      FUEL='PROPANE'/

&DEVC ID='temp_time_history', QUANTITY='TEMPERATURE', XYZ=2.0,2.0,2.0/

&SURF ID='fire',
      COLOR='RED',
      HRRPUA=1000.0,
      TMP_FRONT=300.0/

&OBST ID='Obstruction', XB=0.0,4.0,0.0,4.0,0.0,0.2, RGB=255,255,255, TRANSPARENCY=0.298039/ 
&OBST ID='fire', XB=1.4,2.6,1.4,2.6,0.2,0.4, SURF_IDS='fire','INERT','INERT'/ 

&VENT ID='Mesh Vent: Mesh01 [XMAX]', SURF_ID='OPEN', XB=4.0,4.0,0.0,4.0,0.0,4.0, COLOR='INVISIBLE'/ 
&VENT ID='Mesh Vent: Mesh01 [XMIN]', SURF_ID='OPEN', XB=0.0,0.0,0.0,4.0,0.0,4.0, COLOR='INVISIBLE'/ 
&VENT ID='Mesh Vent: Mesh01 [YMAX]', SURF_ID='OPEN', XB=0.0,4.0,4.0,4.0,0.0,4.0, COLOR='INVISIBLE'/ 
&VENT ID='Mesh Vent: Mesh01 [YMIN]', SURF_ID='OPEN', XB=0.0,4.0,0.0,0.0,0.0,4.0, COLOR='INVISIBLE'/ 
&VENT ID='Mesh Vent: Mesh01 [ZMAX]', SURF_ID='OPEN', XB=0.0,4.0,0.0,4.0,4.0,4.0, COLOR='INVISIBLE'/ 
&VENT ID='Mesh Vent: Mesh01 [ZMIN]', SURF_ID='OPEN', XB=0.0,4.0,0.0,4.0,0.0,0.0, COLOR='INVISIBLE'/ 

&TAIL /

I get the same HRR as HRR_in_full_volume as HRRPUV_in_full_volumne_by_volume_mean*64. But HRRPUV gives very different result. Can someone please what is wrong in approach or understanding?
 Thanks
![0](https://github.com/firemodels/fds/assets/146167548/0e646f70-de2c-4cc8-b397-efe7ec980542)
![1](https://github.com/firemodels/fds/assets/146167548/8a4768d2-355f-439f-84ce-2e133ba8b220)

**Comment by drjfloyd** [comment]
The last device does not use a SPATIAL_STATISTIC. FDS is treating it as a single point measurement made at the center of the XB.

**Comment by Ranhill** [comment]
Thanks, dr_jfloyd!
